app/views/components/tag.njk

```diff
@@ -0,0 +1,3 @@
+{% macro tag(qaIdentifier, text, classes) %}
+  <div class="bc-c-tag {{ classes }}" data-test-id="{{qaIdentifier}}-tag">{{ text }}</div>
```

Comment:
OK I think I know what you mean by qaIdentifier now. The way we have been doing this is the type first and then the identifier. Would you be ok to change this to that style?

Reply:
This will be irrelevant once it is in the component library (I'm assuming we are following the pattern from the nhsuk-frontend) but can change for now

Reply:
just had a look and I think the type is the same as qaIdentifier in this case

Reply:
I will change it to dataTestIdIdentifier